brianlong/www.validators.app
# 184654842 - reorder homepage panels

#### What's this PR do?
PR applies changes to the home page, according to the attached screenshot (in ticket).

#### How should this be manually tested?
Verify the homepage components. Switch between all networks. Test mobile screens.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184654842)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
